physics/forkable_body.hpp

```diff
@@ -153,6 +153,16 @@ not_null<Tr4jectory*> Forkable<Tr4jectory, It3rator>::root() {
 }
 
 template<typename Tr4jectory, typename It3rator>
+Tr4jectory const* Forkable<Tr4jectory, It3rator>::parent() const {
+  return parent_;
```

Comment:
Well then I just need to change the return type to not_null; the question is whether we want parent to fail on a root, or to be optional.